### PR TITLE
update build to use KoreBuild

### PR DESCRIPTION
**Pull request by chenkennt**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿<Project>
+  <Import
```
Comment (xscript) [inline Directory.Build.props]: This can be removed since `AspNetCoreSettings` is not related here.

File: Directory.Build.props [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))\AspNetCoreSettings.props"
+    Condition=" '$(CI)' != 'true' AND '$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))' != '' " />
+
+  <Import Project="version.props" />
+  <Import Project="build\dependencies.props" />
+  <Import Project="build\sources.props" />
+
+  <PropertyGroup>
```
Comment (xscript) [inline Directory.Build.props]: Need to set `projectIconUrl`, otherwise the package can't be published to NuGet.

File: build/dependencies.props [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <RepositoryRoot>$(MSBuildThisFileDirectory)\..</RepositoryRoot>
+  </PropertyGroup>
+  <PropertyGroup Label="Package Versions">
+    <MicrosoftAzureWebJobsPackageVersion>3.0.0-beta8</MicrosoftAzureWebJobsPackageVersion>
```
Comment (xscript) [inline build/dependencies.props]: nit: Better to sort them in alphabetic order for easier maintenance.

File: global.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{ "sdk": {
```
Comment (xscript) [inline global.json]: nit: format

Comment (chenkennt) [reply global.json]: Looks like it's regenerated every time a build is triggered so I'll just leave it as is.

**Pull request merged**